pkgs/by-name/ni/nix-search-cli/package.nix

```diff
@@ -1,33 +1,46 @@
 { lib
 , buildGoModule
 , fetchFromGitHub
-, unstableGitUpdater
 }:
 
-buildGoModule {
+let
+  # latest release version, without "v" prefix and "+commit" build metadata
+  # semver information:
+  # https://github.com/peterldowns/nix-search-cli/releases/tag/v0.2%2Bcommit.7d6b4c5
+  version = "0.2.0";
+  commit = "7d6b4c5";
+  srcHash = "sha256-0Zms/QVCUKxILLLJYsaodSW64DJrVr/yB13SnNL8+Wg=";
+  vendorHash = "sha256-RZuB0aRiMSccPhX30cGKBBEMCSvmC6r53dWaqDYbmyA=";
```

Comment:
to come up with these, i replaced them with `lib.fakeHash` and then ran 

```
NIX_PATH=nixpkgs=/Users/pd/code/nixpkgs nix-build -E 'with import <nixpkgs> {}; callPackage ./package.nix {}'
```
from inside the `pkgs/by-name/ni/nix-search-cli/` directory. Each time the build failed with a mismatched hash error, I replaced the `lib.fakeHash` with whatever hash it expected.